Gemfile.lock

```diff
@@ -121,36 +122,35 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    capistrano (3.14.0)
+    capistrano (3.14.1)
       airbrussh (>= 1.0.0)
       i18n
       rake (>= 10.0.0)
       sshkit (>= 1.9.0)
-    capistrano-bundler (1.6.0)
+    capistrano-bundler (2.0.1)
```

Comment:
### [capistrano-bundler](https://github.com/capistrano/bundler)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 1.6.0 → 2.0.1

[_[change-log](https://github.com/capistrano/bundler/releases), ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/capistrano/bundler/compare/v1.6.0...v2.0.1).

These are the individual commits:
- (da307dc) [Add changelog_uri to metadata to easily link from rubygems.o…](https://github.com/capistrano/bundler/commit/da307dc40a9d761d1911076cfd22f14ec45e515c)
- (b207db4) [Happy New Year! 🥳](https://github.com/capistrano/bundler/commit/b207db407740b86318cd5cd3e4cf8c3cc7ce43b0)
- (d785960) [Upgrade CI to use Ruby 2.7.1 and Bundler 2.1 (#120)](https://github.com/capistrano/bundler/commit/d785960d82bb41c796dffdb910eb28ca86425bfb)
- (4a1bb75) [Maintain release notes using GitHub Actions (#124)](https://github.com/capistrano/bundler/commit/4a1bb75feca2903ab0444baf84d4c355fdbfda2a)
- (148b31f) [Add explicit bundler:config step to support 2.1+ (#122)](https://github.com/capistrano/bundler/commit/148b31fd361a464c9261779e0f98ffacc2132605)
- (b284410) [Release 2.0.0](https://github.com/capistrano/bundler/commit/b284410fbc10d153d8b0f528e6ed74c93a036e69)
- (018b6c4) [Separate default :bundle_without value with : (#126)](https://github.com/capistrano/bundler/commit/018b6c45f75cf78ba500f90cf69e9378a2c74f41)
- (2e875bf) [Release 2.0.1](https://github.com/capistrano/bundler/commit/2e875bfb736d5462a4a67eb3b9edda23b166c2eb)


</details>